api/contract/cli.yml

```diff
@@ -3,123 +3,119 @@ info:
   title: Subset of Influx API covered by Influx CLI
   version: 2.0.0
 servers:
-  - url: /api/v2
```

Comment:
I think this change needs to be made in the upstream openapi repo first - we want to minimize divergence.

Reply:
Should we maybe just have a separate API spec for the legacy API instead of trying to put them together?

Reply:
I'd be interested in thoughts from @jstirnaman as well